test/post_processors/test_collector.py

```diff
@@ -0,0 +1,180 @@
+import pytest
+from curvature.collector import WayCollector
+
+class MockOSMParser(object):
+
+    nodes_callback = None
+    ways_callback = None
+    relations_callback = None
+    coords_callback = None
+
+    nodes = []
+    ways = []
+    relations = []
+    coords = []
+
+    def __init__(self, concurrency=None, nodes_callback=None, ways_callback=None,
```

Comment:
You are providing a lot more implementation then needed to solve the tests below